ONSdigital/rm-case-service
# Stop warning on state change for seft redownload

# What and why?

When a user downloads a SEFT instrument from NOT_STARTED, it goes to INPROGRESS and there is an info level log line.

When a user downloads the instrument a second time, it goes from INPROGRESS to INPROGRESS (which is right as the state hasn't really changed) but theres a warning level log line as that state transition isn't in the state machine despite it being a completely valid action. This PR fixes that so the log line that gets generated is at info level because it's a valid thing to do.

# How to test?

- Run SEFT acceptance tests
- Download SEFT file (notice NOT_STARTED to INPROGRESS)
- Download the file again, it should warn saying INPROGRESS to INPROGRESS via the COLLECTION_INSTRUMENT_DOWNLOADED event isn't allowed.
- Deploy PR to your environment
- Download it again.  Notice how the log line is at level info as it's an accepted event.

# Trello
